crates/fj-core/src/validate/sketch.rs

```diff
@@ -129,4 +172,79 @@ mod tests {
 
         Ok(())
     }
+
+    #[test]
+    fn should_find_clockwise_exterior_cycle() -> anyhow::Result<()> {
+        let mut core = Core::new();
+
+        let valid_outer_circle =
+            HalfEdge::circle([0., 0.], 1., &mut core).insert(&mut core);
+        let valid_exterior =
+            Cycle::new(vec![valid_outer_circle.clone()]).insert(&mut core);
+        let valid_sketch =
+            Sketch::new(vec![
+                Region::new(valid_exterior.clone(), vec![]).insert(&mut core)
+            ]);
+        valid_sketch.validate_and_return_first_error()?;
+
+        let invalid_outer_circle = HalfEdge::from_sibling(
+            &valid_outer_circle,
+            Vertex::new().insert(&mut core),
+        )
+        .insert(&mut core);
+        let invalid_exterior =
+            Cycle::new(vec![invalid_outer_circle.clone()]).insert(&mut core);
+        let invalid_sketch =
+            Sketch::new(vec![
+                Region::new(invalid_exterior.clone(), vec![]).insert(&mut core)
+            ]);
```

Comment:
This can probably be shortened by updating the valid sketch. Something like this:
``` rust
let invalid_sketch = valid_sketch.update_region(valid_sketch.regions().first(), |region|
    region.update_exterior(|cycle| cycle.reverse())
);
```

I didn't test this, so this might not work precisely like this. But something along those lines should be possible.